src/base/projects/Header.svelte

```diff
@@ -23,7 +23,12 @@
 
   // Switches between the browser and commit view
   const switchContent = () => {
-    content = content == ProjectContent.Browser ? ProjectContent.Commits : ProjectContent.Browser;
+    projectStore.update(project => ({ ...project, content: project.content === ProjectContent.History ? ProjectContent.Tree : ProjectContent.History }));
+  };
```

Comment:
Hmm I think this function should take a `ProjectContent` as parameter, and toggle it. The logic should be:

*if the current content equals the one passed in, set to history (default)*
*else, set it to the content passed in*

I'd also name it `toggleContent`

Reply:
done in c1e2717cae73295799262c6a6b02e52d988653c4